### ISSUE TITLE
Guinea-pigging MRGT - Clarifications needed

### ISSUE THREAD

**Issue by RieksJ** [trustoverip/ctwg-toolkit-mrg#27]
The following texts are too unclear for me to understand:

- [ ] line 76 says: "Terminologies are developed and shared on GitHub". Does this mean that all curated texts are drawn from a github repo (implying that I first need to commit stuff and sync repos before I can generate an MRG)? Or only the stuff that is imported from other scopedirs?
- [ ] Please specify the permissions that the github access token must come with (e.g. all repo access?) An image could be helpful
- [ ] lines 126-131 document the configuration, but it is not explicit enough (for me). Please provide concrete examples for all the stuff within `<` and `>` (that of course won't work, but they give me an idea of what it actually is I need to fill in there). I added some examples, but @sih: can you check and make sure they're ok?.

**Comment by sih** [comment]
Just to let you know, I'm off work for a few days so won't be able to get to this until the end of the week

**Comment by RieksJ** [comment]
I either cannot get the stuff filled in properly, or something else is wrong, because my localhost doesn't find a server at port 8083

**Comment by RieksJ** [comment]
Ah the server seems to be running, but I get a 404 error:

![image](https://user-images.githubusercontent.com/8522589/195106815-d30741a3-a254-4856-8a5c-7502073295f3.png)

**Comment by RieksJ** [comment]
Ah. Got it (I hadn't read the documentation): I need to go to `http://localhost:8083/ctwg/mrg` instead of `http://localhost:8083`. Would it be possible to have `http://localhost:8083` work also?

**Comment by sih** [comment]
> The following texts are too unclear for me to understand:
> 
> * [ ]  line 76 says: "Terminologies are developed and shared on GitHub". Does this mean that all curated texts are drawn from a github repo (implying that I first need to commit stuff and sync repos before I can generate an MRG)? Or only the stuff that is imported from other scopedirs?

Yes, this is stated in the first sentence of the _Generating a machine-readable glossary_ section, namely: _"In order to generate the MRG you will have a scope repository on GitHub containing the Scope
Administration File (SAF) and the curated texts."_



> * [ ]  Please specify the permissions that the github access token must come with (e.g. all repo access?) An image could be helpful

Yes good idea have added an image 

> * [ ]  lines 126-131 document the configuration, but it is not explicit enough (for me). Please provide concrete examples for all the stuff within `<` and `>` (that of course won't work, but they give me an idea of what it actually is I need to fill in there). I added some examples, but @sih: can you check and make sure they're ok?.

I have made some minor changes to your texts


> * [ ]  further down I noticed that `http://localhost:8083` doesn't work whereas `http://localhost:8083/ctwg/mrg` does. It should have been a different issue, but to not get things too complicated, here's my request: Can you make `http://localhost:8083` work as if `http://localhost:8083/ctwg/mrg` were typed?

Correct it doesn't work. I was going to reserve the default address for an HTML version of the instructions once they are stable enough (i.e. I don't have to maintain a markdown and an HTML copy). I think this is actually a better idea than just going straight to the form. The form can just be a link from that page.
